### PR TITLE
[EICNET-1881]fix: Replace translation + addresses label + autocomplete internal link

### PR DESCRIPTION
**Pull request by Dromadoros**
How to test it:

- [x] Edit an organisation
- [x] Add social links
- [x] Try to put an internal link and check if auto suggestion is working
- [x] Put multiple addresses and verify that in the front-end we see correct title in top of each address item (it should be company or city)
- [x] Add just 1 location, now it's a paragraph and verify that in front end it's says: 1 location
- [x] Add another location and check if you see: 2 locations

### CODE REVIEW

**Comment by lramarojaona** [comment]
@Dromadoros , for this one: _Try to put an internal link and check if auto suggestion is working_, I guess you're talking about the Link field?

File: lib/modules/eic_private_message/src/Plugin/EntityReferenceSelection/MembershipSelection.php [hunk 1]
```diff
@@ -28,6 +27,36 @@ class MembershipSelection extends DefaultSelection {
...
+
+      // Adding the permission check is sadly insufficient for users: core
+      // requires us to also know about the concept of 'blocked' and 'active'.
+      if (!$this->currentUser->hasPermission('administer users')) {
+        $query->condition('status', 1);
+      }
+
+      return $query;
+    }
+
```
Comment (lramarojaona) [inline lib/modules/eic_private_message/src/Plugin/EntityReferenceSelection/MembershipSelection.php:59]: @Dromadoros , does this have anythin to do with this PR?

**Review by lramarojaona — APPROVED** [review]
It's working, I only have 2 questions

**Comment by Dromadoros** [comment]
> @Dromadoros , for this one: _Try to put an internal link and check if auto suggestion is working_, I guess you're talking about the Link field?

Yes that's the one

File: lib/modules/eic_private_message/src/Plugin/EntityReferenceSelection/MembershipSelection.php (hunk 1 above)
Comment (Dromadoros) [reply lib/modules/eic_private_message/src/Plugin/EntityReferenceSelection/MembershipSelection.php:59]: Yes, it's that one change that fix the issue with link internal suggestions

**Pull request merged**